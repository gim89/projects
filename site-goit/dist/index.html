<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <meta http-equiv="X-UA-Compatible" content="ie=edge">

        <link rel="stylesheet" href="css/bootstrap.css">
    	<link rel="stylesheet" href="css/styles.css">

        <title>Home</title>
    </head>
    <body>
        <div class="container">
            <header class="header">
                <div class="header-section">
                    <a class="header-logo" href="index.html">Journey</a>
                </div>
                <ul class="header-list">
                    <li class="header-item">
                        <a class="header-link" href="index.html">Home</a>
                    </li>
                    <li class="header-item">
                        <a class="header-link" href="about.html">About</a>
                    </li>
                    <li class="header-item">
                        <a class="header-link" href="gallery.html">Gallery</a>
                    </li>
                    <li class="header-item">
                            <a class="header-link" href="tours.html">Tours</a>
                    </li>
                    <li class="header-item">
                            <a class="header-link" href="blog.html">Blog</a>
                    </li>
                    <li class="header-item">
                        <a class="header-link" href="contacts.html">Contacts</a>
                    </li>
                </ul>
            </header>
            <main class="main">
                <div class="slider">
                    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
                        <ol class="carousel-indicators">
                            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
                            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
                            <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
                            <li data-target="#carouselExampleIndicators" data-slide-to="3"></li>
                        </ol>
                        <div class="carousel-inner" role="listbox">
                            <div class="carousel-item active">
                                <picture class="d-block img-fluid">
                                    <source media="(min-width: 1024px)" srcset="img/slider-one-desktop.jpg">
                                    <source media="(min-width: 768px)" srcset="img/slider-one-tablet.jpg">
                                    <img src="img/slider-one-mobile.jpg" alt="First slide">
                                </picture>
                                <div class="carousel-caption d-none d-md-block">
                                    <p class="slider-text">There are plenty of places</p>
                                    <h3 class="slider-title">That are worth seeing</h3>
                                </div>
                            </div>
                            <div class="carousel-item">
                                <picture class="d-block img-fluid">
                                    <source media="(min-width: 1024px)" srcset="img/slider-two-desktop.jpg">
                                    <source media="(min-width: 768px)" srcset="img/slider-two-tablet.jpg">
                                    <img src="img/slider-two-mobile.jpg" alt="Second slide">
                                </picture>
                                <div class="carousel-caption d-none d-md-block">
                                    <p class="slider-text">There are plenty of places</p>
                                    <h3 class="slider-title">That are worth seeing</h3>
                                </div>
                            </div>
                            <div class="carousel-item">
                                <picture class="d-block img-fluid">
                                    <source media="(min-width: 1024px)" srcset="img/slider-three-desktop.jpg">
                                    <source media="(min-width: 768px)" srcset="img/slider-three-tablet.jpg">
                                    <img src="img/slider-three-mobile.jpg" alt="Third slide">
                                </picture>
                                <div class="carousel-caption d-none d-md-block">
                                    <p class="slider-text">There are plenty of places</p>
                                    <h3 class="slider-title">That are worth seeing</h3>
                                </div>
                            </div>
                            <div class="carousel-item">
                                <picture class="d-block img-fluid">
                                    <source media="(min-width: 1024px)" srcset="img/slider-four-desktop.jpg">
                                    <source media="(min-width: 768px)" srcset="img/slider-four-tablet.jpg">
                                    <img src="img/slider-four-mobile.jpg" alt="Forth slide">
                                </picture>
                                <div class="carousel-caption d-none d-md-block">
                                    <p class="slider-text">There are plenty of places</p>
                                    <h3 class="slider-title">That are worth seeing</h3>
                                </div>
                            </div>
                        </div>
                        <div class="controls">
                            <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
                                <svg class="arrow-left">
                                    <use href="img/icons.svg#left"></use>
                                </svg>
                            </a>
                            <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
                                <svg class="arrow-right">
                                    <use href="img/icons.svg#right"></use>
                                </svg>
                            </a>
                        </div>
                    </div>
                </div>
                <section class="destinations">
                    <h2 class="destinations-title">Top destinations</h2>
                    <div class="destinations-card">
                        <div class="card-item">
                            <picture class="d-block img-fluid">
                                <source media="(min-width: 1024px)" srcset="img/card-one-desktop.jpg">
                                <source media="(min-width: 768px)" srcset="img/card-one-tablet.jpg">
                                <img src="img/card-one-mobile.jpg" alt="First card">
                            </picture>
                            <div class="card-item-hover">
                                <h3 class="card-title">Venice</h3>
                                <p class="card-text">
                                    Lorem ipsum dolor site geril amet,
                                    consectetur cing eliti. Suspendisse
                                    nulla leo mew dignissim eu velite ahome/
                                    rew qw vehicula lacinia arcufasecro.
                                    Aenean lacinia ucibusy fase tortor ut
                                    feugiat. Rabitur oliti aliquam bibendum
                                    olor quis malesuadivamu.
                                </p>
                                <a class="btn card-btn" href="#">Details</a>
                            </div>
                        </div>
                        <div class="card-item">
                            <picture class="d-block img-fluid">
                                <source media="(min-width: 1024px)" srcset="img/card-two-desktop.jpg">
                                <source media="(min-width: 768px)" srcset="img/card-two-tablet.jpg">
                                <img src="img/card-two-mobile.jpg" alt="Second slide">
                            </picture>
                            <div class="card-item-hover">
                                <h3 class="card-title">New York</h3>
                                <p class="card-text">
                                    Psum dolor sit ametylo gerto consectetur
                                    ertori hykill holit adipiscing lity. Donecto
                                    rtopil pellentesque osueremo dasi massa eget
                                    cotyll kollit asec emmodem auctor geteq tiloli.
                                    Aliquam dapibus neclol nami wertoli elittro eget
                                    vulpoli no utaterbil congue turpis in suscipit
                                    mintyulitiam ac.
                                </p>
                                <a class="btn card-btn" href="#">Details</a>
                            </div>
                        </div>
                        <div class="card-item">
                            <picture class="d-block img-fluid">
                                <source media="(min-width: 1024px)" srcset="img/card-three-desktop.jpg">
                                <img src="img/card-three-mobile.jpg" alt="Third slide">
                            </picture>
                            <div class="card-item-hover">
                                <h3 class="card-title">Paris</h3>
                                <p class="card-text">
                                    Lorem ipsum dolor site geril amet,
                                    consectetur cing eliti. Suspendisse
                                    nulla leo mew dignissim eu velite a
                                    rew qw vehicula lacinia arcufasecro.
                                    Aenean lacinia ucibusy fase tortor ut
                                    feugiat. Rabitur oliti aliquam bibendum
                                    olor quis malesuadivamu.
                                </p>
                                <a class="btn card-btn" href="#">Details</a>
                            </div>
                        </div>
                    </div>
                </section>
                <div class="home-section">
                    <div class="tabs">
                        <div class="tabs-section">
                            <ul class="nav nav-tabs" role="tablist">
                                <li class="nav-item">
                                    <a class="nav-link active" data-toggle="tab" href="#home" role="tab">
                                        <h4 class="nav-title">Last minute</h4>
                                    </a>
                                </li>
                                <li class="nav-item">
                                    <a class="nav-link" data-toggle="tab" href="#profile" role="tab">
                                        <h4 class="nav-title">Hot deals</h4>
                                    </a>
                                </li>
                                <li class="nav-item">
                                    <a class="nav-link" data-toggle="tab" href="#messages" role="tab">
                                        <h4 class="nav-title">All-inclusive</h4>
                                    </a>
                                </li>
                            </ul>
                            <div class="tab-content">
                                <div class="tab-pane active" id="home" role="tabpanel">
                                    <div class="tab-post">
                                        <picture class="tab-img d-block img-fluid">
                                            <source media="(min-width: 1024px)" srcset="img/tabs-one-desktop.jpg">
                                            <source media="(min-width: 768px)" srcset="img/tabs-one-tablet.jpg">
                                            <img src="img/tabs-one-mobile.jpg" alt="First tabs">
                                        </picture>
                                        <div class="tab-info">
                                            <div class="article-post">
                                                <h5 class="tab-title">Rem psum dr sit </h5>
                                                <p class="tab-passage">
                                                    <a class="tab-link" href="#">Click here</a>
                                                    for more info about this free website template
                                                    created by TemplateMonster.com
                                                </p>
                                                <p class="tab-passage">
                                                    Nulla facilisi. Ut ut tincidunt lacus, ut auctor
                                                    libero. Duis ommodo vel ipsum sed volutpat. Phasellus
                                                    a sagittis dui, eu adipiscinget nisiestibulum eutro.
                                                </p>
                                            </div>
                                            <a class="btn tab-btn" href="#">Details</a>
                                        </div>
                                    </div>
                                    <div class="tab-post">
                                        <picture class="tab-img d-block img-fluid">
                                            <source media="(min-width: 1024px)" srcset="img/tabs-four-desktop.jpg">
                                            <source media="(min-width: 768px)" srcset="img/tabs-four-tablet.jpg">
                                            <img src="img/tabs-four-mobile.jpg" alt="Fourth tabs">
                                        </picture>
                                        <div class="tab-info">
                                            <div class="article-post">
                                                <h4 class="tab-title">Hem psuf abr sit </h4>
                                                <p class="tab-passage">
                                                    Julla facilisi. Ut ut tincidunt lacus, ut auctor
                                                    libero. Fuis ommodo vel ipsum sed volutpat. Phasellus
                                                    a sagittis dui, eu adipiscinget nisi. Vestibulum eu
                                                    eleifend metus, ut ornare nibh. Vestibulumul tincidunt
                                                    interdum libero vitae faucibus. Gonec dapibus feugiate
                                                    auctor. In ac dapibus lacus. Maecenas in pharetra mim
                                                    asellus a sagittis dui, eu adipiscinget nisi.
                                                </p>
                                            </div>
                                            <a class="btn tab-btn" href="#">Details</a>
                                        </div>
                                    </div>
                                </div>
                                <div class="tab-pane" id="profile" role="tabpanel">
                                    <div class="tab-post">
                                        <picture class="tab-img d-block img-fluid">
                                            <source media="(min-width: 1024px)" srcset="img/tabs-two-desktop.jpg">
                                            <source media="(min-width: 768px)" srcset="img/tabs-two-tablet.jpg">
                                            <img src="img/tabs-two-mobile.jpg" alt="Second tabs">
                                        </picture>
                                        <div class="tab-info">
                                            <div class="article-post">
                                                <h4 class="tab-title">Rem psum dr sit </h4>
                                                <p class="tab-passage">
                                                    <a class="tab-link" href="#">Click here</a>
                                                    for more info about this free website template
                                                    created by TemplateMonster.com
                                                </p>
                                                <p class="tab-passage">
                                                    Nulla facilisi. Ut ut tincidunt lacus, ut auctor
                                                    libero. Duis ommodo vel ipsum sed volutpat. Phasellus
                                                    a sagittis dui, eu adipiscinget nisiestibulum eutro.
                                                </p>
                                            </div>
                                            <a class="btn tab-btn" href="#">Details</a>
                                        </div>
                                    </div>
                                    <div class="tab-post">
                                        <picture class=" tab-img d-block img-fluid">
                                            <source media="(min-width: 1024px)" srcset="img/tabs-five-desktop.jpg">
                                            <source media="(min-width: 768px)" srcset="img/tabs-five-tablet.jpg">
                                            <img src="img/tabs-five-mobile.jpg" alt="Fifth tabs">
                                        </picture>
                                        <div class="tab-info">
                                            <div class="article-post">
                                                <h4 class="tab-title">Hem psuf abr sit </h4>
                                                <p class="tab-passage">
                                                    Julla facilisi. Ut ut tincidunt lacus, ut auctor
                                                    libero. Fuis ommodo vel ipsum sed volutpat. Phasellus
                                                    a sagittis dui, eu adipiscinget nisi. Vestibulum eu
                                                    eleifend metus, ut ornare nibh. Vestibulumul tincidunt
                                                    interdum libero vitae faucibus. Gonec dapibus feugiate
                                                    auctor. In ac dapibus lacus. Maecenas in pharetra mim
                                                    asellus a sagittis dui, eu adipiscinget nisi.
                                                </p>
                                            </div>
                                            <a class="btn tab-btn" href="#">Details</a>
                                        </div>
                                    </div>
                                </div>
                                <div class="tab-pane" id="messages" role="tabpanel">
                                    <div class="tab-post">
                                        <picture class=" tab-img d-block img-fluid">
                                            <source media="(min-width: 1024px)" srcset="img/tabs-three-desktop.jpg">
                                            <source media="(min-width: 768px)" srcset="img/tabs-three-tablet.jpg">
                                            <img src="img/tabs-three-mobile.jpg" alt="Third tabs">
                                        </picture>
                                        <div class="tab-info">
                                            <div class="article-post">
                                                <h4 class="tab-title">Rem psum dr sit </h4>
                                                <p class="tab-passage">
                                                    <a class="tab-link" href="">Click here</a>
                                                    for more info about this free website template
                                                    created by TemplateMonster.com
                                                </p>
                                                <p class="tab-passage">
                                                    Nulla facilisi. Ut ut tincidunt lacus, ut auctor
                                                    libero. Duis ommodo vel ipsum sed volutpat. Phasellus
                                                    a sagittis dui, eu adipiscinget nisiestibulum eutro.
                                                </p>
                                            </div>
                                            <a class="btn tab-btn" href="*">Details</a>
                                        </div>
                                    </div>
                                    <div class="tab-post">
                                        <picture class="tab-img d-block img-fluid">
                                            <source media="(min-width: 1024px)" srcset="img/tabs-six-desktop.jpg">
                                            <source media="(min-width: 768px)" srcset="img/tabs-six-tablet.jpg">
                                            <img src="img/tabs-six-mobile.jpg" alt="Six tabs">
                                        </picture>
                                        <div class="tab-info">
                                            <div class="article-post">
                                                <h4 class="tab-title">Hem psuf abr sit </h4>
                                                <p class="tab-passage">
                                                    Julla facilisi. Ut ut tincidunt lacus, ut auctor
                                                    libero. Fuis ommodo vel ipsum sed volutpat. Phasellus
                                                    a sagittis dui, eu adipiscinget nisi. Vestibulum eu
                                                    eleifend metus, ut ornare nibh. Vestibulumul tincidunt
                                                    interdum libero vitae faucibus. Gonec dapibus feugiate
                                                    auctor. In ac dapibus lacus. Maecenas in pharetra mim
                                                    asellus a sagittis dui, eu adipiscinget nisi.
                                                </p>
                                            </div>
                                            <a class="btn tab-btn" href="#">Details</a>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="newsletter">
                        <h4 class="newsletter-title">Newsletter</h4>
                        <div class="newsletter-body">
                            <form class="newsletter-form" action="#">
                                <label for="email">Sign up to receive our newsletters</label>
                                    <input id="email" type="email" placeholder="email address">
                            </form>
                            <ul class="newsletter-list">
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Fgo psu dr sit</a></li>
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Sem psum dr sit ametre</a></li>
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Rame sum dr sit ame</a></li>
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Bem psum dr sit ameteko</a></li>
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Nem dsum dr sit</a></li>
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Vcem psum dr</a></li>
                                <li class="item newsletter-item"><a class="newsletter-link" href="#">Zdfem psum dr sittr</a></li>
                            </ul>
                        </div>
                    </div>
                </div>
            </main>
            <footer class="footer">
                <div class="footer-part">
                    <ul class="footer-list">
                        <li class="footer-item">
                            <a class="footer-link" href="#">faqs page</a>
                        </li>
                        <li class="footer-item">
                            <a class="footer-link" href="#">people say</a>
                        </li>
                    </ul>
                    <ul class="footer-list">
                        <li class="footer-item">
                            <a class="footer-link" href="#">useful links</a>
                        </li>
                        <li class="footer-item">
                            <a class="footer-link" href="#">partners</a>
                        </li>
                    </ul>
                    <ul class="footer-list">
                        <li class="footer-item">
                            <a class="footer-link" href="#">insurance</a>
                        </li>
                        <li class="footer-item">
                            <a class="footer-link" href="#">family travel</a>
                        </li>
                    </ul>
                    <div class="footer-contact">
                        <h4 class="footer-title">Contact us:</h4>
                        <p class="footer-text">
                            <i class="footer-picture">
                                <svg class="footer-img">
                                    <use href="img/icons.svg#smartphone"></use>
                                </svg>
                            </i>
                            +1-800-234-5678
                        </p>
                        <p class="footer-text">
                            <i class="footer-picture">
                                <svg class="footer-img">
                                            <use href="img/icons.svg#letter"></use>
                                </svg>
                            </i>
                            info@denfiks.org
                        </p>
                    </div>
                </div>
                <div class="footer-part">
                    <ul class="social">
                        <li class="social-item">
                            <a class="social-link" href="#">
                                <svg class="social-img twitter">
                                    <use href="img/icons.svg#twitter"></use>
                                </svg>
                            </a>
                        </li>
                        <li class="social-item">
                            <a class="social-link" href="#">
                                <svg class="social-img facebook">
                                    <use href="img/icons.svg#facebook"></use>
                                </svg>
                            </a>
                        </li>
                        <li class="social-item">
                            <a class="social-link" href="#">
                                <svg class="social-img google">
                                    <use href="img/icons.svg#google"></use>
                                </svg>
                            </a>
                        </li>
                        <li class="social-item">
                            <a class="social-link" href="#">
                                <svg class="social-img instagram">
                                    <use href="img/icons.svg#linkedin"></use>
                                </svg>
                            </a>
                        </li>
                    </ul>
                    <div>
                        <p class="footer-copyrighted">
                            Journey &copy; 2013 | Privacy Policy | Website designed by TemplateMonster.com
                        </p>
                    </div>
                </div>
            </footer>
        </div>
        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
    </body>
</html>