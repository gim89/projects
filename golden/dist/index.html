<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=940">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>golden</title>
  <link rel="stylesheet" href="css/styles.css">
</head>
<body>
  <header class="head">
    <div class="container">
      <nav class="nav">
        <a href="" class="nav__logo">Golden</a>
        <ul class="nav__list">
          <li class="nav__item nav__item--hover">
            <a href="#" class="nav__link">home</a>
          </li>
          <li class="nav__item nav__item--hover">
            <a href="#" class="nav__link">services</a>
          </li>
          <li class="nav__item nav__item--hover">
            <a href="#" class="nav__link">portfolio</a>
          </li>
          <li class="nav__item nav__item--hover">
            <a href="#" class="nav__link">about</a>
          </li>
          <li class="nav__item nav__item--hover">
            <a href="#" class="nav__link">contact</a>
          </li>
        </ul>
      </nav>
      <div class="head__promo">
        <h2 class="head__title">Welcome To Our Studio!</h2>
        <h1 class="head__caption">It`s nice to meet you</h1>
        <a href="##" class="head__btn btn btn--hover btn--active">Tell me more</a>
      </div>
    </div>
  </header>
  <section class="service section__padding">
    <div class="container">
      <h2 class="service__title">service</h2>
      <p class="service__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
      <div class="service__body">
        <div class="service__item">
          <img src="img/service-one.jpg" alt="" class="service__img">
          <h3 class="service__caption">E-Commerce</h3>
          <p class="service__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Asperiores vel accusamus commodi.</p>
        </div>
        <div class="service__item">
          <img src="img/service-two.jpg" alt="" class="service__img">
          <h3 class="service__caption">Responsive Web</h3>
          <p class="service__text">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nam voluptatem dolorem nisi.</p>
        </div>
        <div class="service__item">
          <img src="img/service-three.jpg" alt="" class="service__img">
          <h3 class="service__caption">Web Security</h3>
          <p class="service__text">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi sapiente deleniti fugiat!</p>
        </div>
      </div>
    </div>
  </section>
  <section class="portfolio section__padding">
    <div class="container">
      <h2 class="portfolio__title">our portfolio</h2>
      <p class="portfolio__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
      <div class="portfolio__body">
        <div class="portfolio__item">
          <img src="img/portfolio-one.jpg" alt="" class="portfolio__img">
          <h3 class="portfolio__caption">Ebony Ivory</h3>
          <p class="portfolio__text">Branding</p>
        </div>
        <div class="portfolio__item">
          <img src="img/portfolio-two.jpg" alt="" class="portfolio__img">
          <h3 class="portfolio__caption">Smart Stationary</h3>
          <p class="portfolio__text">Print Design</p>
        </div>
        <div class="portfolio__item">
          <img src="img/portfolio-three.jpg" alt="" class="portfolio__img">
          <h3 class="portfolio__caption">Clever Poster</h3>
          <p class="portfolio__text">Print Design</p>
        </div>
        <div class="portfolio__item">
          <img src="img/portfolio-four.jpg" alt="" class="portfolio__img">
          <h3 class="portfolio__caption">Vinyl Record</h3>
          <p class="portfolio__text">Product Mock-Up</p>
        </div>
        <div class="portfolio__item">
          <img src="img/portfolio-five.jpg" alt="" class="portfolio__img">
          <h3 class="portfolio__caption">Treehouse Template</h3>
          <p class="portfolio__text">Web Design</p>
        </div>
        <div class="portfolio__item">
          <img src="img/portfolio-six.jpg" alt="" class="portfolio__img">
          <h3 class="portfolio__caption">Burned Logo</h3>
          <p class="portfolio__text">Branding</p>
        </div>
      </div>
    </div>
  </section>
  <section class="about section__padding">
    <div class="container">
      <h2 class="about__title">about us</h2>
      <p class="about__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
      <div class="about__body">
        <div class="about__item">
          <h3 class="about__caption">July 2010</h3>
          <p class="about__subcaption">Our Humble Beginnings</p>
          <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
        </div>
        <div class="about__item">
          <img src="img/about-one.jpg" alt="" class="about__img">
        </div>
        <div class="about__item"></div>
        <div class="about__item"></div>
        <div class="about__item">
          <img src="img/about-two.jpg" alt="" class="about__img">
        </div>
        <div class="about__item">
          <h3 class="about__caption">January 2011</h3>
          <p class="about__subcaption">Facing Startup Battles</p>
          <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
        </div>
        
        
        <div class="about__item">
          <h3 class="about__caption">December 2012</h3>
          <p class="about__subcaption">Enter The Dark Days</p>
          <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
        </div>
        <div class="about__item">
          <img src="img/about-three.jpg" alt="" class="about__img">
        </div>
        <div class="about__item"></div>
        <div class="about__item"></div>
        <div class="about__item">
          <img src="img/about-four.jpg" alt="" class="about__img">
        </div>
        <div class="about__item">
          <h3 class="about__caption">February 2014</h3>
          <p class="about__subcaption">Our Triumph</p>
          <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
        </div>
        <div class="about__item"></div>
        <div class="about__item">
          <button class="about__btn btn--hover btn--active">our story continues ...</button>
        </div>
        <div class="about__item"></div>
      </div>
    </div>
  </section>
  <section class="team section__padding">
    <div class="container">
      <h2 class="team__title">our amazing team</h2>
      <p class="team__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
      <div class="team__body">
        <div class="team__item">
          <img src="img/team.jpg" alt="" class="team__img">
          <h3 class="team__caption">Kimberly Thompson</h3>
          <p class="team__span">Marketer</p>
          <div class="team__social">
            <div class="twitter twitter--hover social__icon"></div>
            <div class="facebook facebook--hover  social__icon"></div>
            <div class="pinterest pinterest--hover  social__icon"></div>
          </div>
        </div>
        <div class="team__item">
          <img src="img/team.jpg" alt="" class="team__img">
          <h3 class="team__caption">Rico Massimo</h3>
          <p class="team__span">Coder</p>
          <div class="team__social">
            <div class="twitter twitter--hover social__icon"></div>
            <div class="facebook facebook--hover social__icon"></div>
            <div class="google google--hover social__icon"></div>
          </div>
        </div>
        <div class="team__item">
          <img src="img/team.jpg" alt="" class="team__img">
          <h3 class="team__caption">Uku Mason</h3>
          <p class="team__span">Graphic Designer</p>
          <div class="team__social">
            <div class="facebook facebook--hover social__icon"></div>
            <div class="pinterest pinterest--hover social__icon"></div>
          </div>
        </div>
      </div>
      <p class="team__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium excepturi, dignissimos tempore nisi et maxime officiis tempora? Eligendi, architecto minus?</p>
    </div>
  </section>
  <div class="logos">
    <div class="container logos__container">
      <img src="img/logos-one.jpg" alt="" class="logos__img">
      <img src="img/logos-two.jpg" alt="" class="logos__img">
      <img src="img/logos-three.jpg" alt="" class="logos__img">
      <img src="img/logos-four.jpg" alt="" class="logos__img">
    </div>
  </div>
  <section class="contact section__padding">
    <div class="container">
      <h2 class="contact__title">about us</h2>
      <p class="subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
      <form action="" class="contact__form">
        <div class="contact__item">
          <div class="contact__inputs">
            <input type="text" class="contact__name" placeholder="your name*" required>
            <input type="email" class="contact__email" placeholder="your e-mail*" required>
            <input type="text" class="contact__subject" placeholder="subject">
          </div>
          <textarea name="" class="contact__message" placeholder="your message"></textarea>
        </div>
        <div class="contact__item">
            <input type="submit" class="contact__btn btn btn--hover btn--active" value="Send Message">
        </div>
      </form>
    </div>
  </section>
  <footer class="footer-page">
    <div class="container footer-page__container">
      <p class="footer-page__text">&copy; Copyright 2014 FreebiesXpress.com</p>
      <div class="footer-page__social">
          <a href="#" class="social__icon twitter twitter--hover footer-page__icon"></a>
          <a href="#" class="social__icon facebook facebook--hover footer-page__icon"></a>
          <a href="#" class="social__icon pinterest pinterest--hover footer-page__icon"></a>
          <a href="#" class="social__icon google google--hover footer-page__icon"></a>
      </div>
    </div>
  </footer>
</body>
</html>